### PR TITLE
0.3 integration bug fixes

### PR DESCRIPTION
**Pull request by matejonnet**

### CODE REVIEW

File: pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ public BuildDriver getBuildDriver(BuildType buildType) throws CoreException {
             }
         }
 
-        throw new CoreException("No build driver available for " + buildType + " build type.");
+        throw new CoreException("No build driver available for " + buildType + " build type."); //TODO create test to make sure exception is properly handled
```
Comment (slaskawi) [inline pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java]: Todo to be removed

File: pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java [hunk 2]
```diff
@@ -1,23 +1,5 @@
 package org.jboss.pnc.core.builder;
```
Comment (slaskawi) [inline pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java]: This change shouldn't be there

**Comment by slaskawi** [comment]
retest this please

File: pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java (hunk 1 above)
Comment (matejonnet) [inline pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java]: Why do you think it's wrong if TODO is left there, we are not GA yet? Before removing it we need to make sure that it is done. Do you think we should create jira for it? I think Jira ticket has to much overhead for such task.

Comment (slaskawi) [inline pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java]: I think we should either implement this test straight away (my preference) or if we can't do that for some reason - we should create a JIRA for that - not a TODO in the code. Without JIRA we won't be able to track it.

File: pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java (hunk 2 above)
Comment (matejonnet) [inline pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java]: That's an issue because IDEs are automatically arranging imports. We need need to share the same formatter settings to avoid it.

Comment (slaskawi) [inline pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java]: I don't think that commiting a file without meaningful changes is a good practice. It adds only a noise.

File: pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java (hunk 1 above)
Comment (matejonnet) [inline pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java]: This is opening wider discussion that is out of scope of this PR, do we allow TODOs in commited code? Maybe we need a Jira to track the discussion ;)

**Comment by matejonnet** [comment]
Retest this please.

**Comment by matejonnet** [comment]
Retest this please.

**Comment by slaskawi** [comment]
retest this please
